inference-engine/src/cldnn_engine/cldnn_engine.cpp

```diff
@@ -291,6 +291,8 @@ InferenceEngine::CNNNetwork clDNNEngine::CloneAndTransformNetwork(const Inferenc
             manager.register_pass<ngraph::pass::UnrollTensorIterator>();
             manager.run_passes(nGraphFunc);
         }
+    } else {
```

Comment:
I suppose this is a dead code because of https://github.com/openvinotoolkit/openvino/pull/3729/files#diff-2068b2d1a649108c988c3ac72a911ecbcffabd68900214582cc8b5d50c6dd251R125, is not it?

Reply:
I think will be better to have a double check, because we can break this code if change something in `cloneNetwork` method.

Moreover if we sure that `cloneNetwork` throws an exception and don't want to have a double check, we can remove ` if (clonedNetwork.getFunction()) {`

Reply:
Actually we have this check a bit later in Program c-tor:
https://github.com/openvinotoolkit/openvino/blob/241b0faea1d49c5b9ded9c17181d102afe07397f/inference-engine/src/cldnn_engine/cldnn_program.cpp#L103-L105

So If `clonedNetwork.getFunction()` returns nullptr here, then this method won't do anything else, and then exception will be thrown from Program c-tor.